inventory/hosts.localhost

```diff
@@ -0,0 +1,29 @@
+#bare minimum hostfile
+
+[OSEv3:children]
+masters
+nodes
+etcd
+
+[OSEv3:vars]
+ansible_user=root
+# if ansible_user is not root you must set the following
+#ansible_become=yes
```

Comment:
Lines 9-11 can be removed due to the requirement of either running as root or running ansible-playbook using sudo.